Panic parsing go-pe's yaml

I'm looking to add support for parsing the export table to go-pe but when trying to run binparsergen against the YAML (without any modifications) I get the following panic

```
PS C:\Users\schmi\source\go-pe> binparsegen.exe .\conversion.spec.yaml
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x526a14]

goroutine 1 [running]:
www.velocidex.com/golang/binparsergen.GenerateProfileCode(0xc0000b0380, 0x9, 0xc0000b69c0, 0x2, 0x2)
        C:/Users/schmi/source/binparsergen/profile.go:50 +0x2e4
www.velocidex.com/golang/binparsergen.GenerateCode(0xc000092180, 0xc0000b69c0, 0x5769e5, 0x7)
        C:/Users/schmi/source/binparsergen/compile.go:50 +0x14c
main.main()
        C:/Users/schmi/source/binparsergen/cmd/binparsegen.go:26 +0x16
```

This is because the parser for various structures returns a nil value for fields that I believe are not included due to the whitelist in the YAML.

--- a/profile.go
+++ b/profile.go
@@ -44,6 +44,9 @@ func GenerateProfileCode(
 		struct_name = NormalizeName(struct_name)
 		for _, field_name := range struct_def.fields {
 			field_def := struct_def.Fields[field_name]
+			if field_def == nil {
+				continue
+			}
 
 			result += fmt.Sprintf("    Off_%s_%s int64\n",
 				struct_name, field_name)

--- a/struct.go
+++ b/struct.go
@@ -30,6 +30,10 @@ func (self *%[1]s) Size() int {
 
 	for _, field_name := range definition.fields {
 		field_def := definition.Fields[field_name]
+		if field_def == nil {
+			continue
+		}
+
 		result += field_def.GetParser().Compile(name, field_name)
 	}
 
@@ -42,6 +46,10 @@ func GenerateDebugString(name string, profile_name string, definition *StructDef
 			"\"struct %s @ %%#x:\\n\", self.Offset)\n", name, name)
 	for _, field_name := range definition.fields {
 		field_def := definition.Fields[field_name]
+		if field_def == nil {
+			continue
+		}
+
 		if field_def.StringParser != nil ||
 			field_def.UTF16StringParser != nil {
 			result += fmt.Sprintf(

